www/doc/en/dev_env.md

```diff
@@ -10,7 +10,8 @@ A web server is necessary to test the scripts locally while developing. Any web
 server that can serve files with the brython_directory/www as document root will 
 work ; you can use the built-in web server provided in the distribution : open 
 a console window, go to the directory, and run `python server.py`. This will 
-start the server on port 8000 (edit _server.py_ to change the port number).
+start the server on port 8000 (if you want to use other port number you can use, 
```

Comment:
I recommend stating it this way:
start the server on port 8000 (if you want to use a different port number you can use,